src/ecdc/ecdc.c

```diff
@@ -825,6 +835,26 @@ ecdc_alloc_list_command(struct ecdc_console * console,
 }
 
 
+char const * ecdc_replace_prompt(struct ecdc_console *console, char const *prompt)
```

Comment:
Update format to be consistent with the rest of the file
```
char const *
ecdc_replace_prompt(struct ecdc_console *console, char const *prompt)
{
    // ....
}
```

Reply:
Done.